packages/firestore/lite/test/integration.test.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+
+import { initializeApp } from '@firebase/app-exp';
+import { getFirestore, initializeFirestore } from '../src/api/database';
+
+describe('Firestore', () => {
+  it('can provide setting', () => {
+    const app = initializeApp(
+      { apiKey: 'fake-api-key', projectId: 'test-project' },
+      'test-app-initializeFirestore'
+    );
+    initializeFirestore(app, { host: 'localhost', ssl: false });
```

Comment:
Did you forget to assert that the setting object on the Firestore instance equals the input?

Reply:
I only meant to make sure that this doesn't throw. There is no API to read the settings. I added an instanceOf check to make this more like a proper test.